@ecomplus/storefront-components/src/js/TheAccount.js

```diff
@@ -127,18 +106,40 @@ export default {
   },
 
   created () {
-    this.navTabs = [this.i19registration, this.i19orders, this.i19favorites]
+    this.navTabs = [
+      {
+        label: this.i19registration,
+        value: 'account'
+      },
+      {
+        label: this.i19orders,
+        value: 'orders'
+      },
+      {
+        label: this.i19favorites,
+        value: 'favorites'
+      }
+    ]
     const { favorites } = this.ecomPassport.getCustomer()
     this.favoriteIds = favorites || []
     if (this.ecomPassport.checkAuthorization()) {
       this.ecomPassport.requestApi('/orders.json?transactions.type=recurrence&limit=1&fields=_id')
         .then(({ data }) => {
           const { result } = data
           if (result.length) {
-            this.navTabs.push(this.i19subscriptions)
+            this.navTabs.push({
+              label: this.i19subscriptions,
+              value: 'subscriptions'
+            })
           }
         })
         .catch(console.error)
     }
+    if (Array.isArray(this.customer.loyalty_points_entries) && this.customer.loyalty_points_entries.length) {
```

Comment:
```suggestion
    if (this.customer.loyalty_points_entries && this.customer.loyalty_points_entries.length) {
```

Só pra encurtar seu if, não tem como isso existir e não ser um array...

Reply:
Nunca vai desrespeitar o schema da API